decidim-proposals/app/forms/decidim/proposals/admin/proposals_fork_form.rb

```diff
@@ -37,7 +37,11 @@ def mergeable_to_same_component
             !proposal.official? || proposal.votes.any? || proposal.endorsements.any?
           end
 
-          errors.add(:proposal_ids, :public_proposals) if public_proposals
+          if public_proposals
```

Comment:
💬 Would it be clear for the user if you do the add the errors inside the previous loop depending on the detected problems? 

You can use a set to avoid adding the same error twice, like this:
```ruby
errors_set = Set[]
proposals.each do |proposal|
  errors_set << :published if proposal.published? # this was not checked before, I don't know the exact logic
  errors_set << :official if proposal.official?
  errors_set << :supported if proposal.votes.any? || proposal.endorsements.any?
end

errors_set.each {|error| errors.add(:base, error)} if errors_set.any?
```

Reply:
Good point, I'll make the changes you suggest, I'll ping you when ready

Reply:
I've updated the messages and removed the published? condition, I've been looking for a reason for this condition and I haven't found anything. I've also inverted the validation message about official proposals.